admin/data/sql/manual/20190814161826.do.draft-v2-vewpupilregister.sql

```diff
@@ -0,0 +1,31 @@
+CREATE VIEW [mtc_admin].[vewPupilRegisterV2_Preview] AS
+    SELECT
+     p.id as pupilId,
+         p.foreName,
+         p.middleNames,
+         p.lastName,
+         p.urlSlug,
+         p.dateOfBirth,
+         p.school_id,
+         g.id as groupId,
+         ISNULL(g.name, '-') as groupName,
```

Comment:
Why default to dash and not allow nulls on db layer? The UI could display what is appropriate based on null values.

Reply:
i have no idea George, this is preserved from the original query.  I assumed you wrote it 😄